docs/docsite/rst/dev_guide/testing.rst

```diff
@@ -68,7 +68,6 @@ When Shippable detects an error and it can be linked back to a file that has bee
 
    The test `ansible-test sanity --test validate-modules` failed with the following errors:
```

Comment:
```suggestion
   The test `ansible-test sanity --test validate-modules` failed with the following error:
```

Reply:
leaving preexisting text w/o change, iirc it does show multiple errors